src/Illuminate/Foundation/helpers.php

```diff
@@ -586,8 +586,10 @@ function mix($path, $manifestDirectory = '')
             $manifestDirectory = "/{$manifestDirectory}";
         }
 
-        if (file_exists(public_path($manifestDirectory.'/hot'))) {
-            $url = rtrim(file_get_contents(public_path($manifestDirectory.'/hot')));
+        $hotFile = public_path($manifestDirectory.'/hot');
+
+        if (file_exists($hotFile)) {
```

Comment:
Why not inline it ?
```php
if (file_exists($hotFile = public_path($manifestDirectory.'/hot'))) {
    $url = rtrim(file_get_contents($hotFile));
```

Reply:
I I thought about this:
```php
        if (file_exists(public_path($manifestDirectory.'/hot'))) {
            $url = rtrim(file_get_contents(public_path($manifestDirectory.'/hot')), "/ \t\n\r\0\x0B");
```
There is no unnecessary `rtrim`, the variable is not used (in most cases hot reloading is not used, while a variable is always assigned). But it seemed to me too ugly. Hard choice )